supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesTask.scala

```diff
@@ -45,9 +45,12 @@ final case class TesTask(jobDescriptor: BackendJobDescriptor,
     Option(commandScriptContents)
   )
 
-  private val commandScriptOut = commandScript.copy(
-    url = Option(tesPaths.script.toString),
-    contents = None
+  private val commandScriptOut = Output(
+    Option("commandScript"),
```

Comment:
I actually quite like the `paramName = ...` format for long lists of arguments, so I'd advocate for bring it back it possible

Reply:
I brought that format back here and added it where relevant elsewhere.